ansible/vagrant/README.md

```diff
@@ -20,7 +20,7 @@ vagrant plugin install vagrant-openstack-provider --plugin-version ">= 0.6.1"
 Vagrant uses Ansible to automate the Kubernetes deployment. Install Ansible (Mac OSX example):
 ```
 sudo easy_install pip
-sudo pip install ansible
+sudo pip install ansible==2.0.0.2
```

Comment:
@eparis @rutsky we need to pin the version in this case. The default Ansible version (2.0.1.0 for OSX 10.10.5) barfs when running the cluster pb for coreos. This is a common OSX version and I can see many new dev's runnin into the same issue. I can create a seperate issue on why the coreos pb's don't work with ansible 2.0.1.0